lib/nfg_onboarder/onboarding_controller_helper.rb

```diff
@@ -161,6 +161,8 @@ def on_before_show
       def on_valid_step
         onboarding_session.current_high_level_step = controller_name
         onboarding_session.current_step = next_step
+        onboarding_session.owner = get_event_target
+        onboarding_session.name = get_onboarder_name if get_event_target.present?
         update_onboarding_session_step_data
```

Comment:
I'm nervous about how this change will effect existing onboarders. I think it would be better to keep the onboarding functionality the same and just change the get_onboarder_name method in the import_data_controller in the importer gem to supply the name as it is expected.